include/deal.II/base/array_view.h

```diff
@@ -618,6 +640,8 @@ make_array_view (ElementType *const begin, ElementType *const end)
 {
   Assert(begin <= end,
          ExcMessage("The beginning of the array view should be before the end."));
+  Assert(internal::ArrayViewHelper::is_contiguous(begin, end),
+         ExcMessage("The provided range isn't contiguous in memory!"));
```

Comment:
I don't think you need to test here -- you give two pointers, of course the range is contiguous (by definition).

Reply:
I agree.